### PR TITLE
PhpdocToCommentFixer - also check other control structures, besides foreach

### PR DESCRIPTION
**Pull request by ceeram**
See #1034

### CODE REVIEW

File: Symfony/CS/Fixer/Symfony/PhpdocToCommentFixer.php [hunk 1]
```diff
@@ -41,7 +41,14 @@ public function fix(\SplFileInfo $file, $content)
...
 
-            if ($nextToken->isGivenkind(T_FOREACH) && $this->isValidForeach($tokens, $token, $nextIndex)) {
+            static $controlStructures = array(
+                T_FOREACH,
+                T_IF,
+                T_SWITCH,
+                T_WHILE,
+                T_FOR,
+            );
+            if ($nextToken->isGivenkind($controlStructures) && $this->isValidControl($tokens, $token, $nextIndex)) {
```
Comment (GrahamCampbell) [inline Symfony/CS/Fixer/Symfony/PhpdocToCommentFixer.php]: We normally have an extra new line before opening an if statament.

**Comment by sstok** [comment]
Seems good to me :+1:

File: Symfony/CS/Fixer/Symfony/PhpdocToCommentFixer.php (hunk 1 above)
Comment (ceeram) [inline Symfony/CS/Fixer/Symfony/PhpdocToCommentFixer.php]: changed

**Comment by GrahamCampbell** [comment]
:+1:

**Comment by stof** [comment]
Please also add tests for the ``list($a, $b)`` syntax to assign variables (and also inside foreach loops for PHP 5.5+)

**Comment by stof** [comment]
:+1:

**Comment by keradus** [comment]
Thank you @ceeram.

**Pull request closed without merging**